assets/js/views/edit.js

```diff
@@ -0,0 +1,38 @@
+var EditView = Backbone.View.extend({
+  template: AppTemplates.edit,
+
+  events: {
+    'click .delete-button': 'deletePost',
+    'click .save-button': 'savePost'
+  },
+  deletePost: function(ev) {
+    ev.preventDefault();
+
+    this.model.destroy({title: title, body: body});
+  },
+
+  savePost: function(ev) {
+    ev.preventDefault();
+
+    var title = title.$el.find('#title').val();
```

Comment:
This throws an error that `Cannot read property '$el' of undefined`. You want to look up `$el` on the current view.

Right here, you are trying to set the `title` value for the first time so you won't have anything to work with when you then try to say `title.$el` because `title` hasn't yet been set until the full line evaluates.

The same thing applies to `body` below.